jest.config.js

```diff
@@ -0,0 +1 @@
+process.env.TZ = 'UTC';
```

Comment:
This is required to ensure that tests for `now` and `today` don't become timezone-dependent.